paasta_tools/smartstack_tools.py

```diff
@@ -479,6 +481,7 @@ def get_replication_for_instance(
         instance_pool = instance_config.get_pool()
         for location, hosts in attribute_host_dict.items():
             hostname = self._get_first_host_in_pool(hosts, instance_pool)
+            log.debug("Getting replication info from host %s", hostname)
```

Comment:
nit: maybe include the pool (`from host %s for pool %s`) since it will print this message out several times (once per pool)